src/corehost/packaging/BuildToolsVersion.txt

```diff
@@ -1 +0,0 @@
-1.0.25-prerelease-00228-02
```

Comment:
Maybe this is just a bad diff but why are these showing up as deleted instead of moved? These are still necessary.